package.json

```diff
@@ -49,7 +49,7 @@
     "@storybook/addon-actions": "6.2.9",
     "@storybook/addon-console": "1.2.3",
     "@storybook/addon-essentials": "6.2.9",
-    "@storybook/addon-knobs": "6.2.9",
+    "@storybook/addon-knobs": "6.3.1",
```

Comment:
Hey @SusCasasola I would recommend you to check https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-62x-to-630

It might be worth going for Storybook 6.3 instead as it deprecates some addons and I'm not fully sure about compatibility once it comes to Storybook minor upgrades.

Let me know what you think!

Reply:
Thanks a lot @viniciuskneves! You are right, it's seems that they are replacing the knobs with controls, so I will need to update the components that are using knobs.